components/WpImage.vue

```diff
@@ -193,48 +193,49 @@ export default {
             if (this.height) {
                 return parseInt(this.height)
             }
-            return _get(this, "image.mediaDetails.height", "auto")
+            return this.image.mediaDetails?.height || auto
         },
         parsedWidth() {
             // default to defined width
             if (this.width) {
                 return parseInt(this.width)
             }
-            return _get(this, "image.mediaDetails.width", "auto")
+            return this.image.mediaDetails?.width || "auto"
         },
         parsedSrc() {
-            return this.src || _get(this, "image.sourceUrl", "")
+            return this.src || this.image.sourceUrl || ""
         },
         parsedSrcset() {
-            return this.srcset || _get(this, "image.srcSet", "")
+            return this.srcset || this.image.srcSet || ""
```

Comment:
This should be:

```
this.image?.srcSet || ""
```

Reply:
I'm guessing you want me to fix all of these computed items, not tjust the srcSet one.

...like so?

` parsedSizes() {
            return this.sizes || this.image?.sizes || ""
        },
         parsedVideoUrl() {
            return this.videoUrl || this.image?.imageMeta?.videoUrl || ""
        },
I'm going to leave it as it was plus the changes you requested, until tonight to double check with you.

Reply:
Yes please fix all